WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderBlogFragment.java

```diff
@@ -90,7 +90,7 @@ private void checkEmptyView() {
 
         if (hasBlogAdapter() && getBlogAdapter().isEmpty()) {
             actionableEmptyView.setVisibility(View.VISIBLE);
-            actionableEmptyView.image.setImageResource(R.drawable.img_illustration_empty_results_216dp);
+            actionableEmptyView.image.setImageResource(R.drawable.img_illustration_empty_results_162dp);
```

Comment:
The previous illustration was a general purpose empty view illustration used throughout the app. The new illustration seems specific to the following tab so I think we might want to consider using a more specific name. Wdyt?

Reply:
I also think the dimension value in the name usually corresponds to max(width, height). Can you please double check it, thanks!

Reply:
Good suggestion @malinajirka on the name. Changing the remainder of the empty views are slated for a future project. I'm stumped on the size, it appears to use the pattern of max width specified in the vector itself. I've changed to match this pattern, along with renaming the file to reflect the following feature.

Reply:
Updated in 8d07bf2